etc/adt/config.template

```diff
@@ -57,7 +57,7 @@ DEPLOYMENT_ES_IMAGE=${DEPLOYMENT_ES_IMAGE}
 DEPLOYMENT_ES_IMAGE_VERSION=${DEPLOYMENT_ES_IMAGE_VERSION}
 DEPLOYMENT_ES_HEAP=${DEPLOYMENT_ES_HEAP}
 DEPLOYMENT_ONLYOFFICE_HTTP_PORT=${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}
-EXO_HOST_PORT=${EXO_HOST_PORT}
+ONLY_OFFICE_HOST_PORT=${ONLY_OFFICE_HOST_PORT}
```

Comment:
Sorry about that, but I just  realized the variable should be named DEPLOYMENT_ONLY_OFFICE_HOST_PORT to be coherent with the usual naming

Reply:
sounds consistent, updated in code